axelor-account/src/main/java/com/axelor/apps/account/web/InvoiceLineController.java

```diff
@@ -625,4 +626,31 @@ public void manageInvoiceLineAxis(ActionRequest request, ActionResponse response
       TraceBackService.trace(response, e);
     }
   }
+
+  public void checkAnalyticAccount(ActionRequest request, ActionResponse response) {
+    try {
+
+      if (Invoice.class.equals(request.getContext().getContextClass())) {
+        Invoice invoice = request.getContext().asType(Invoice.class);
+        if (invoice != null && CollectionUtils.isNotEmpty(invoice.getInvoiceLineList())) {
+          for (InvoiceLine invoiceLine : invoice.getInvoiceLineList()) {
+            if (invoiceLine != null && invoiceLine.getAccount() != null) {
+              Beans.get(AccountService.class)
```

Comment:
please do not call Beans.get inside a for loop as it will cause performance issues